bundle/config/mutator/python/python_mutator.go

```diff
@@ -152,42 +161,71 @@ func (m *pythonMutator) runPythonMutator(ctx context.Context, cacheDir string, r
 		inputPath,
 		"--output",
 		outputPath,
+		"--diagnostics",
+		diagnosticsPath,
```

Comment:
It doesn't have to be consistent because changes need to be backward and forward-compatible. If we make a breaking change, we raise the minimum required version.

Reply:
Where is the minimum required version stored/communicated?

Reply:
We don't. For now, we can add a minimal required version of CLI as the configuration in the template, and document it.

We can also check in VerifyCliVersion, but right now, it's executed before we run the Python mutator if we want to append a different minimum required version there. Another difficulty is that this configuration parameter might already be used. 

Another way is to pass the current CLI version into Python code and check it there. We can make it a part of the effective bundle configuration.